### PR TITLE
Added button to copy launch arguments easier

### LINKED ISSUE #1158: [FEATURE REQUEST] - Enable right-click or add a "Copy" button for launch parameters.
**What problem does the feature request solve?**
The app as a whole and the "Launching the game from mod manager" section in particular doesn't respond when right-clicked, which makes copying the parameters on a Steam Deck an enormous hassle. Adding a button to copy to clipboard or just enabling right-click would be very helpful.

### PR DESCRIPTION
**Pull request by ebkr**
Resolves #1158 

### Screenshots
image

image

### CODE REVIEW

File: src/pages/Help.vue [hunk 1]
```diff
@@ -49,7 +49,19 @@
                     Your current argument would be:
                     <code v-if="doorstopTarget.length > 0">{{ doorstopTarget }}</code>
                     <code v-else>These parameters will be available after installing BepInEx.</code>
+                    <br/>
                 </p>
+                <br/>
+                <template v-if="doorstopTarget.length > 0">
+                    <p>
+                        <button class="button" @click="copyDoorstopTargetToClipboard" v-if="!copyingDoorstopText">
+                            <i class="fas fa-clipboard" v-tooltip.left="'Drag to reorder'"></i>
```
Comment (anttimaki) [inline src/pages/Help.vue:58]: Says "Drag to reorder". Anyway, I don't see this or the "Copy launch arguments" tooltip at all?

**Review by anttimaki — APPROVED** [review]
Approving as I don't consider it a major issue, but the tooltip doesn't seem to work. Anyway I'd argue that tooltip that says the same as the button itself provides no value so it should be removed altogether.

**Comment by ebkr** [comment]
The tooltip was likely just from a bad copy/paste job on another item that had a tooltip. Will remove.

**Review by anttimaki — APPROVED** [review]
LGTM

**Comment by Guessed8822** [comment]
@ebkr thank you!

**Pull request merged**